src/Api/Releases.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Gitlab API library.
+ *
+ * (c) Matt Humphrey <matth@windsor-telecom.co.uk>
+ * (c) Graham Campbell <hello@gjcampbell.co.uk>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Gitlab\Api;
+
+class Releases extends AbstractApi
+{
+    /**
+     * @param int|string $project_id
+     * @param array $parameters {
+     *
+     * @return mixed
+     *
+     * @var string $sort Return releases sorted in asc or desc order. Default is desc.
```

Comment:
The syntax for this is wrong, and some of the other paramters docs are missing/incorrect, too.